radio/src/gui/480x272/model_select.cpp

```diff
@@ -42,6 +42,7 @@ ModelsList modelslist;
 ModelsCategory * currentCategory;
 int currentCategoryIndex;
 ModelCell * currentModel;
+char modelDeleteConfirm[(2*LEN_MODEL_FILENAME)+3];
```

Comment:
I am not sure the length is good, it would be better to use the 2 length constants here